docs/_ext/intersphinx_custom.py

```diff
@@ -0,0 +1,422 @@
+"""
```

Comment:
@phlax can you describe what your changes are at the top of this comment block? And link to an issue filed to track removing?

Reply:
yep for sure...